tools/wpt/testfiles.py

```diff
@@ -162,9 +162,8 @@ def exclude_ignored(files, ignore_rules):
 
 
 def files_changed(revish, ignore_rules=None, include_uncommitted=False, include_new=False):
-    """Get and return files changed since current branch diverged from master,
-    excluding those that are located within any path matched by
-    `ignore_rules`."""
+    """Get and return files changed since `revish`, excluding those that are
```

Comment:
This isn't quite right; if revish is a range it will return the files changed in that range rather than since that range.

Reply:
Yeah, you're right. It doesn't make sense to have anything but HEAD as the end of the range because affected tests are determined using the checkout (working tree = HEAD + modified + new) but you're still right. I'll revert this and leave it alone, again. What I'd prefer is https://github.com/web-platform-tests/wpt/pull/14093 and changing everything that suggests that revish could be a range to taking about a single revision, and possibly even asserting that it is so even though A..B will happen to work with `git diff`.

Reply:
The original docstring is confusing (inaccurate?), too. If we fail to summarize the behaviour of `git diff` in various different cases, shall we just expose the implementation detail by explicitly saying `git diff [revish]` is what we do under the hood? I think that'd be less bad.